stdlib/public/Darwin/Foundation/Data.swift

```diff
@@ -62,6 +62,39 @@ internal func __NSDataIsCompact(_ data: NSData) -> Bool {
 
 #endif
 
+@_alwaysEmitIntoClient
+internal func _withStackOrHeapBuffer(capacity: Int, _ body: (UnsafeMutableBufferPointer<UInt8>) -> Void) {
```

Comment:
this does not mimic the C behavior that has a larger capacity first off and second off it is missing the jumbo capacity for the main thread (which has a larger stack allocation)

Reply:
Given the benchmarks, what's the rationale for using a large buffer?